src/utils/power_of_two.hh

```diff
@@ -0,0 +1,28 @@
+#ifndef POWER_OF_TWO_HH
+#define POWER_OF_TWO_HH
+
+#include "narrow.hh"
+#include <bit>
+#include <cassert>
+
+namespace openmsx {
+
+template<typename T>
```

Comment:
Just a suggestion: maybe instead of accepting any type `T` you could constrain this template to:
`template<std::unsigned_integral T>`
(and then also `#include <concepts>`).

Reply:
Will do.